doc/integration_test_report.rst

```diff
@@ -343,6 +343,17 @@ Traceability from integration tests to requirements
     :sourcetype: impacts_on
     :stats:
 
+Traceability via intermediate items
+-----------------------------------
+
+.. item-matrix::
+    :source: DESIGN-
+    :target: UTEST ITEST
```

Comment:
You are missing intermediate flag here

Reply:
I've made it optional. The source and target options are optional as well.

Reply:
Its example of the intermediate matrix so i would assume we want that demonstrated here...

Reply:
Yes, I do. I'll add it. 

The problem with this example is that the DESIGN items are linked to mostly top-level RQT items with child RQT items. Those child RQT items are validated by TEST items, but not the top-level RQT items. This results in a rather empty item-matrix. Is this an issue that can and should be addressed?

Reply:
Added intermediate flag in 4bc55e5c68e232847e16b67bc555603102f362b5.